noggin/security/ipa_admin.py

```diff
@@ -6,30 +6,6 @@
 
 class IPAAdmin(object):
 
-    __WRAPPED_METHODS = (
```

Comment:
these methods will still need to be wrapped, because they are performed by the admin account.